src/pages/HomePage/HomePage.js

```diff
@@ -1,14 +1,36 @@
+/* eslint-disable react/jsx-one-expression-per-line */
 import React from 'react';
-import HomePageWrapper from './HomePage.style';
 import Hero from '../../components/Hero/Hero';
-import Info from '../../components/Info/Info';
+import InfoSection from '../../components/InfoSection/InfoSection';
 
 function HomePage() {
   return (
-    <HomePageWrapper>
+    <>
       <Hero />
-      <Info />
-    </HomePageWrapper>
+      <InfoSection name="how-it-works" title="How it works">
```

Comment:
I would wrap these two InfoSection components into a wrapper. The wrapper handles the margin to the heatmap and footer. Then you don't need to hard-code the height. The InfoSection component should get a margin-top or margin-bottom that handles the distance between the InfoSection elements